.github/workflows/compare_translations.py

```diff
@@ -114,6 +134,30 @@ def check_translations():
         sys.exit(0)
 
 
+def main():
+    """
+     Parse command-line arguments, check for the existence of the specified directory,
+    and call check_translations with the provided or default directory.
+
+    """
+    parser = argparse.ArgumentParser(
+        description="Check and print missing translations for all non-default languages."
+    )
+    parser.add_argument(
+        "--directory",
+        type=str,
+        nargs="?",
+        default=os.path.join(os.getcwd(), "lang"),
```

Comment:
This is probably not correct. You probably tested from the root directory. Use the `countline.py` script as an example for this.

Reply:
Please elaborate more what I have to do here ?

Reply:
I may have made a mistake.